01-builder-visualization/README.md

```diff
@@ -19,3 +19,8 @@ CARTO BUILDER for Data Visualization
 * Exercise: [category-bubbles](exercises/category-bubble.md)
```

Comment:
agree, I wasn't sure about which format was better, so I used both and waited for your feedback :)